packages/backend/src/paymentPointerKey/model.ts

```diff
@@ -2,14 +2,13 @@ import { BaseModel } from '../shared/baseModel'
 
 import { JWKWithRequired } from 'auth'
 
-export class ClientKeys extends BaseModel {
+export class PaymentPointerKey extends BaseModel {
   public static get tableName(): string {
-    return 'clientKeys'
+    return 'paymentPointerKeys'
   }
 
-  // The id should be the same as the id in the kid.
   public id!: string
-  public clientId!: string
+  public paymentPointerId!: string
 
   public jwk!: JWKWithRequired
```

Comment:
What do you think about the idea of flattening the jwk?
https://github.com/interledger/rafiki/pull/681#discussion_r1006835706

Reply:
Last time `jwk` had `clientDetails` in it, it was excluded because it made it hard to work with the tools related to jwk. I think it's fine right now without the flattening.